.claude/skills/daytona-sandbox/scripts/probe-sandboxes.sh

```
@@ -0,0 +1,87 @@
+  exit 2
+fi
+
+# ---- Sandboxes -----------------------------------------------------------
+echo "=== Sandboxes ==="
+if ! sb_json=$(daytona sandbox list -f json 2>&1); then
+  echo "  could not list sandboxes:"
+  echo "$sb_json" | sed 's/^/    /'
+  exit 1
+fi
+
+echo "$sb_json" | python3 - <<'PY'
```

Shellcheck (0.11.0)

[error] This redirection overrides piped input. To use both, merge or pass filenames.

(SC2259)